msteams-platform/apps-in-teams-meetings/meeting-app-extensibility.md

```diff
@@ -49,34 +50,73 @@ With the pre-meeting app experience, you can find and add meeting apps and perfo
+
+![Mobile pre-meeting experience](../assets/images/apps-in-meetings/mobilepremeeting.png)
+
+---
+
 #### In-meeting app experience
 
 With the in-meeting app experience, you can engage participants during the meeting by using apps and the in-meeting dialog box. Meeting apps are hosted in the top upper bar of the meeting window as an in-meeting tab. Use the in-meeting dialog box to showcase actionable content for meeting participants. For more information, see [create apps for Teams meetings](create-apps-for-teams-meetings.md).
 
+For mobile, meeting apps are hosted in the ellipses menu &#x25CF;&#x25CF;&#x25CF; in the meeting under **Apps** menu. Selecting the **Apps** menu will show all the apps available in the meeting.
+
 **To use tabs during a meeting**
 
+# [Desktop](#tab/desktop)
+
 1. After entering the meeting, from the top upper bar of the chat window, select the app you want to use. An app is visible in a Teams meeting in the side panel or the in-meeting dialog box.
 1. In the in-meeting dialog box, enter your response as a feedback.
 
     ![Dialog box view](../assets/images/apps-in-meetings/in-meeting-dialog-view.png)
 
-    > [!NOTE]
-    > * Apps can leverage the Teams Client SDK to access the `meetingId`, `userMri`, and `frameContext` to render the experience appropriately.
-    > * If the in-meeting dialog box is rendered successfully, you are notified that the results successfully downloaded.
-    > * Your app manifest specifies the places where you want them to appear. The context field is used for this purpose. It can also be part of a share-tray experience, subject to specified design guidelines.
+# [Mobile](#tab/mobile)
+
+After entering the meeting and adding the app as per steps to use tabs during a meeting from desktop or web, the app is visible in mobile Teams meeting under the **Apps** section. Selecting **Apps** will show the list of apps and user can launch any of the app as an in meeting side panel of the app. User can interact with the app in it and launch it for users.
 
-    The following image illustrates the in-meeting side panel:
+The in-meeting dialog box is displayed where you can enter your response as a feedback.
 
-    ![In-meeting side panel](../assets/images/apps-in-meetings/in-meeting-dialog.png)
+![Mobile dialog box view](../assets/images/apps-in-meetings/mobile-in-meeting-dialog-view.png)
+
+> [!NOTE]
+> You do not require to change the app manifest for the apps to work on mobile.
+
+---
+
+> [!NOTE]
+> * Apps can leverage the Teams Client SDK to access the `meetingId`, `userMri`, and `frameContext` to render the experience appropriately.
+> * If the in-meeting dialog box is rendered successfully, you are notified that the results successfully downloaded.
+> * Your app manifest specifies the places where you want them to appear. The context field is used for this purpose. It can also be part of a share-tray experience, subject to specified design guidelines.
```

Comment:
Your app manifest specifies the places that you want them to appear. The context field is used for this purpose. It is also the part of a share-tray experience, subject to specified design guidelines.